oojs.js

```diff
@@ -4,9 +4,30 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+  var Playlist {
+    constructor(title, songs, favorites){
+      this.title = title;
+      this.songs = songs;
+      this.favorites= 0;
+      }
+    addSong: function() {
+      this.songs ++;
+      return 'whine up!';
+    }
+  }
+
 // Create an instance of the Playlist class.
+  let Dancehall = new Playlist('Danceup', ('Bubble Time', 'Mash up de place', 0));
 
 // Create an Album class that inherits from Playlist. It should also...
   // Have an artist property that is determined by some input.
 
+  class Album extends Playlist {
+    constructor(title, songs, favorites, artist){
+      super(title, songs, favorites)
+      this.artist = artist;
+    }
+  }
+
 // Create an instance of the Album class.
+let Small-Things = new Album('he', ('she', 'it'), 4, Blink 182); 
```

Comment:
`Small-Things`-- javascript will think you're trying to subtract `Things` from `Small`. Try `smallThings`

`['she','it']` careful about mixing up your parens and square brackets.

I'm glad to see you progressing with the tougher JS stuff, test your code and keep pushing yourself! 👍